### ISSUE TITLE
Смешные книги в библиотеке 

### ISSUE THREAD

**Issue by UsadoSmokk** [TauCetiStation/TauCetiClassic#9547]
#### Подробное описание проблемы
Спавнятся книги с ссылкой на вики тау, что по факту делает их кладом для рофлов и мемов
#### Что должно было произойти
Спавнится все что угодно, даже у вуди встала шишка, но не это
#### Что произошло на самом деле
Заспавнилась книга Rules и книга вики тау киты
![image](https://user-images.githubusercontent.com/56721822/176254345-18c2ec41-dbcf-4169-9af7-81e8788adb42.png)
#### Как повторить
Найти и дождаться спавна, наверное
#### Дополнительная информация:
Смешной прикол, мне нравится

**Comment by RichardJones1** [comment]
Эти книги служат для того, чтобы вместо того чтобы кидать незадачливым коллегам ссылки на вики в LOOC - можно было ингейм им вручить книжку с гайдом. А на счет заглавной странички вики - мб какие то ссылки поломались..

**Comment by UsadoSmokk** [comment]
> Эти книги служат для того, чтобы вместо того чтобы кидать незадачливым коллегам ссылки на вики в LOOC - можно было ингейм им вручить книжку с гайдом. А на счет заглавной странички вики - мб какие то ссылки поломались..

гайды и буквально ссылка на вики, где есть все - это разное

**Comment by UsadoSmokk** [comment]
> Насколько помню там не должно быть книг ведущих на заглавную стр
ладно, но книжки с ссылкой на правила точно быть не должно

**Comment by RichardJones1** [comment]
> > Насколько помню там не должно быть книг ведущих на заглавную стр
> > ладно, но книжки с ссылкой на правила точно быть не должно

Согласен, такое должно быть только в арсенале у админов, чтобы можно было заспавнить какому нибудь нарушителю в инвентарь

**Comment by volas** [comment]
это mediawiki убрала печатную версию страницы в последних обновлениях, нужно переверстывать вики как было
